.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["react", "es2015", "stage-0"],
+  "presets": [
+    "react",
+    ["latest", {"es2015": {"modules": false}}],
+    "stage-0"
+  ],
```

Comment:
Oh neat, I did see the env present when I was researching webpacker but latest seems closer to es2015 (it's just a superset), I didn't realise it was deprecated, and I didn't want to do too much in one change.

Maybe we could use latest in this PR and open another for env?